data/mods/Medieval_Stuff/recipe_armor.json

```diff
@@ -0,0 +1,385 @@
+  "time": 540000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "textbook_armwest", 5 ]],
+  "qualities": [
+    {"id":"HAMMER","level":3,"amount":1}
+  ],
+  "tools": [
+    [ [ "tongs", -1 ] ],
+    [ [ "chisel", -1 ] ],
+    [ [ "anvil", -1 ] ],
+    [ [ "crucible", -1 ] ],
+    [
+      [ "forge", 300 ],
+      [ "char_forge", 60 ],
+      [ "oxy_torch", 60 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "steel_lump", 10 ],
+      [ "steel_chunk", 40 ]
+    ],
+    [
+      [ "fur", 4 ],
+      [ "leather", 4 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "chainmail_hauberk",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_SUIT",
+  "skill_used": "tailor",
+  "difficulty": 2,
+  "time": 18000,
+  "reversible": true,
+  "autolearn": true,
+  "components": [
+    [ [ "chainmail_vest", 1 ] ],
+    [ [ "chainmail_arms", 1 ] ],
+    [ [ "chainmail_legs", 1 ] ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "chainmail_suit",
+  "id_suffix": "hauberk",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_SUIT",
+  "skill_used": "tailor",
+  "difficulty": 3,
+  "time": 2000,
+  "reversible": true,
+  "autolearn": true,
+  "components": [
+    [ [ "chainmail_hood", 1 ] ],
+    [ [ "chainmail_hauberk", 1 ] ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "armor_cuirass",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_TORSO",
+  "skill_used": "fabrication",
+  "difficulty": 5,
+  "time": 500000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "textbook_armwest", 8 ]],
+  "qualities": [
+    {"id":"HAMMER","level":3,"amount":1}
+  ],
+  "tools": [
+    [ [ "tongs", -1 ] ],
+    [ [ "crucible", -1 ] ],
+    [
+      [ "forge", 500 ],
+      [ "char_forge", 100 ],
+      [ "oxy_torch", 100 ]
+    ]
+  ],
+  "components": [ [ [ "scrap_bronze", 28 ] ] ]
+},{
+  "type" : "recipe",
+  "result": "armor_lorica",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_TORSO",
+  "skill_used": "fabrication",
+  "difficulty": 7,
+  "time": 550000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "textbook_armwest", 8 ]],
+  "qualities": [
+    {"id":"HAMMER","level":3,"amount":1}
+  ],
+  "tools": [
+    [ [ "tongs", -1 ] ],
+    [ [ "chisel", -1 ] ],
+    [ [ "anvil", -1 ] ],
+    [ [ "swage", -1 ] ],
+    [
+      ["forge", 450],
+      ["char_forge", 90],
+      ["oxy_torch", 90]
+    ]
+  ],
+  "components": [
+    [ [ "steel_lump", 15 ] ],
+    [
+      [ "fur", 12 ],
+      [ "tanned_pelt", 2 ],
+      [ "leather", 12 ],
+      [ "tanned_hide", 2 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "shendyt",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_LEGS",
+  "skill_used": "tailor",
+  "difficulty": 3,
+  "time": 90000,
+  "reversible": false,
+  "autolearn": true,
+  "tools": [
+    [
+      [ "needle_bone", 30 ],
+      [ "needle_wood", 30 ],
+      [ "tailors_kit", 30 ],
+      [ "sewing_kit", 30 ]
+    ]
+  ],
+  "components": [ [ [ "rag", 16 ] ] ]
```

Comment:
16 rags for something like this?
Also, shouldn't kilts be fancy? They're useless from practical point of view.